backend-kotlin/src/main/proto/payment.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
you could use the proto file that is in the proto folder of the repo and remove this file